addons/mail/static/src/components/chat_window_header/chat_window_header.scss

```diff
@@ -54,6 +54,11 @@
     text-overflow: ellipsis;
 }
 
+.o_Clickable_ChatWindowHeaderName:hover {
+    font-weight: bold;
+    text-decoration: underline;
+}
+
```

Comment:
Those are style change, should be below the style category, positioned alphabetically with other class in there.